projects/cardiovascular-patient-app/src/app/report-BP/report-BP.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit } from '@angular/core';
+import { AbstractControl, FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'report-BP',
+  templateUrl: './report-BP.component.html',
+  styleUrls: ['./report-BP.component.css'],
+})
+export class ReportBPComponent implements OnInit {
+  submitted = false;
+
+  form: FormGroup = this.fb.group({
```

Comment:
```suggestion
  form = this.fb.group({
```
so Typescript will infer the correct type